src/fifelse.c

```diff
@@ -42,57 +42,78 @@ SEXP fifelseR(SEXP l, SEXP a, SEXP b) {
   const int *restrict pl = LOGICAL(l);
   SEXP ans = PROTECT(allocVector(TYPEOF(a), len0)); nprotect++;
   copyMostAttrib(a, ans);
+  
+  if(!isNull(na))
+  {
```

Comment:
please try to align to existing coding style of that file, so the following
```c
  if(!isNull(na))
  {
```
turns into
```c
  if (!isNull(na)) {
```